packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx

```diff
@@ -54,14 +54,3 @@ export function createStorageBrowser<
 
   return { StorageBrowser };
 }
-
-// below is functioning as a "type canary", will start yelling if `elements`
-// inference is broken
-const {
-  StorageBrowser: {
-    LocationsView: { Controls },
-  },
-} = createStorageBrowser({ elements: { Button } });
-const _SillyTest = () => (
-  <Controls.Refresh.Button alignContent={'-moz-initial'} />
-);
```

Comment:
```suggestion
```

Reply:
Updated in latest to fix merge conflicts